docs/learn/smart-contract-developers/mint-lsp7-token.md

```diff
@@ -8,196 +8,89 @@ import TabItem from '@theme/TabItem';
 
 # Mint LSP7 Token
 
-:::caution Disclaimer
+In this guide you will mint some [LSP7 Digital Asset](../../standards/tokens/LSP7-Digital-Asset.md) tokens to the EOA that controls the contract.
 
-This guide might contain outdated information and will be updated soon.
+## Setup
 
-:::
-
-This guide will teach you how to mint some [LSP7 Digital Asset](../../standards/tokens/LSP7-Digital-Asset.md) tokens to your [Universal Profile](../../standards/universal-profile/lsp0-erc725account.md).
-
-## Mint tokens for your Universal Profile
-
-The code snippet below shows how to mint 100 tokens with your Universal Profile as a beneficiary.
+The following code snippets require to install a few libraries.
 
-Make sure you have the following dependencies installed before beginning this tutorial:
-
-- Either [`web3.js`](https://github.com/web3/web3.js) or [`ethers.js`](https://github.com/ethers-io/ethers.js/)
+- [`ethers.js`](https://github.com/ethers-io/ethers.js/)
 - [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts/)
 
-<Tabs>
-  
-  <TabItem value="web3js" label="web3.js">
-
-```shell title="Install the dependencies"
-npm install web3 @lukso/lsp-smart-contracts
-```
-
-  </TabItem>
-
-  <TabItem value="ethersjs" label="ethers.js">
-
-```shell title="Install the dependencies"
+```shell
 npm install ethers @lukso/lsp-smart-contracts
 ```
 
-  </TabItem>
-
-</Tabs>
-
-### Step 1 - Setup imports and constants
+### Imports and constants
 
-At this point you will need a private key in order to mint some tokens as well as the `LSP7Mintable` _token contract address_.
-We will import `LSP7Mintable` in order to get the _ABI_ of the contract that we will interact with.
-
-<Tabs>
-  
-  <TabItem value="web3js" label="web3.js">
-
-```javascript
-import LSP7Mintable from '@lukso/lsp-smart-contracts/artifacts/LSP7Mintable.json';
-import Web3 from 'web3';
-
-const web3 = new Web3('https://rpc.testnet.lukso.network');
-const privateKey = '0x...';
-const myTokenAddress = '0x...';
-const universalProfileAddress = '0x...';
-
-// setup your EOA
-const myEOA = web3.eth.accounts.wallet.add(privateKey);
-```
-
-  </TabItem>
-
-  <TabItem value="ethersjs" label="ethers.js">
+At this point, the `LPS7Mintable` contract is being prepared for the following interaction. You construct an instance of a contract, using its _ABI_ and the _contract address_.
```

Comment:
```suggestion
After defining the core parameters of the [`LPS7 Mintable`](../../contracts/contracts/LSP7DigitalAsset/presets/LSP7Mintable.md) contract, you are able to create an instance using its _ABI_ and the _contract address_.
```
What does "the following interaction" mean? Is it necessary. The above suggestion would be my approach.